msccl/language/mscclpp/ir.py

```diff
@@ -256,7 +266,14 @@ def remove_empty_fields(d):
                     )
                     i_buff = {"src": op.src.buffer.value, "dst": op.dst.buffer.value}
                     dsts = list(map(lambda x: {"buff": x.buffer.value, "off": x.index}, op.dsts))
-                elif op.inst == Instruction.copy or op.inst == Instruction.copy_packet:
+                elif (
+                    op.inst == Instruction.copy
+                    or op.inst == Instruction.copy_packet
+                    or op.inst == Instruction.transform_to_packet
+                ):
+                    src = op.src
+                    dst = op.dst
+                elif op.inst == Instruction.transform_to_packet:
```

Comment:
Seems to be transform from packet instead?

Reply:
Thanks fixed